test/test-zero-len-file.sh

```diff
@@ -50,7 +50,8 @@ begin_test "pull zero len file"
   clone_repo "$reponame" clone
   rm clone.log
 
-  git status | grep "working directory clean"
+  git status 2>&1 | tee status.log
+  grep "working (directory|tree) clean" status.log
```

Comment:
@sinbad @rubyist: thanks for the `grep -E` reminder, I totally forgot 😁 

As for the pipe, I added this since @technoweenie pointed it out, but I neglected to run the tests because I was getting up to grab a glass of water. The `| tee` actually fails the test, since it plops a file in the working tree, causing the status to be unclean!

I removed the `tee` pipe and fixed the `grep -E` situation in https://github.com/github/git-lfs/pull/1369/commits/52c77362895735d32c6a9cbf53a20a2ed494997e